test/test_arraycontext.py

```diff
@@ -307,6 +312,23 @@ def get_imag(ary):
                 # can't do in place operations with a scalar lhs
                 continue
 
+            if op_func == operator.ge:
+                op_func_actx = actx.np.greater_equal
+            elif op_func == operator.lt:
+                op_func_actx = actx.np.less
+            elif op_func == operator.gt:
+                op_func_actx = actx.np.greater
+            elif op_func == operator.eq:
+                op_func_actx = actx.np.equal
+            elif op_func == operator.ne:
+                op_func_actx = actx.np.not_equal
+            elif op_func == get_real:
+                op_func_actx = actx.np.real
+            elif op_func == get_imag:
+                op_func_actx = actx.imag
```

Comment:
How come pytato doesn't support these as is? Should be straightforward AFAICT...

Reply:
At least the comparison operators: pytato arrays cannot support those.

Reply:
removed real(), imag() from this list, left the comparison operators.